risc0/zkvm/prove/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "risc0-zkvm-prove"
+version = "0.1.0"
+edition = "2021"
+links = "risc0-zkvm-prove"
+
+[dependencies]
+anyhow = "1.0"
```

Comment:
👍 to "anyhow" which looks super cool to make error handling less cumbersome.  I was looking at "quick_error", but this looks somewhat easier to use.